httpclient/src/test/kotlin/tbdex/sdk/httpclient/E2ETest.kt

```diff
@@ -0,0 +1,233 @@
+package tbdex.sdk.httpclient
+
+import tbdex.sdk.httpclient.models.ErrorResponse
+import tbdex.sdk.httpclient.models.GetExchangesFilter
+import tbdex.sdk.httpclient.models.GetExchangesResponse
+import tbdex.sdk.httpclient.models.GetOfferingsFilter
+import tbdex.sdk.httpclient.models.GetOfferingsResponse
+import tbdex.sdk.httpclient.models.SendMessageResponse
+import tbdex.sdk.httpclient.models.TbdexResponse
+import tbdex.sdk.protocol.models.Message
+import tbdex.sdk.protocol.models.Offering
+import tbdex.sdk.protocol.models.Order
+import tbdex.sdk.protocol.models.OrderStatus
+import tbdex.sdk.protocol.models.Quote
+import tbdex.sdk.protocol.models.Rfq
+import tbdex.sdk.protocol.models.RfqData
+import tbdex.sdk.protocol.models.SelectedPaymentMethod
+import web5.sdk.credentials.VerifiableCredential
+import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.dids.DidKey
+import kotlin.test.Ignore
+import kotlin.test.Test
+
+private const val pollInterval = 1000L
+
+class E2ETest {
+  /**
+   * An e2e test calling a tbdex client to perform a tbdex transaction.
+   *
+   */
+  @Test
+  @Ignore
```

Comment:
intentionally ignoring?

Reply:
yes. i didn't want this to run as part of the CI, because this test only works if i have tbdex-mock-pfi running at the same time.